src/app/components/Feedback/Feedback.jsx

```diff
@@ -1,62 +1,107 @@
+/* global alert */
 import React from 'react';
 import PropTypes from 'prop-types';
 import FocusTrap from 'focus-trap-react';
+import axios from 'axios';
 
 import { trackDiscovery } from '../../utils/utils';
+import appConfig from '../../data/appConfig';
+
+const initialFields = {
+  Email: '',
+  Feedback: '',
+};
 
 class Feedback extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = { showForm: false };
+    this.currentURL = (this.props.location.pathname +
+    this.props.location.hash +
+    this.props.location.search);
```

Comment:
Nitpick: this syntax is weird.

Reply:
Refactored, because the URL was not updating correctly. Now, `Application` handles the URL to avoid re-rendering `Feedback` unnecessarily.